config/command/pom.xml

```diff
@@ -53,18 +53,15 @@
     <dependencies>
         <dependency>
             <groupId>org.osgi</groupId>
-            <artifactId>osgi.core</artifactId>
-            <scope>provided</scope>
```

Comment:
Why remove 'provided' scope?

Reply:
The declaration in bom specifies scope=provided, so all users get that for free.

Reply:
@rovarga ok, the bom already defines `provided` by default. Got it. Thanks.